app/lib/widgets/ToDoTaskItem.dart

```diff
@@ -49,11 +49,9 @@ class _ToDoTaskItemState extends State<ToDoTaskItem> {
   @override
   void initState() {
     super.initState();
-    _avatars = _getMockAvatars(countPeople);
+    avatars = getMockAvatars(countPeople);
     if (widget.dateTime.contains('All Day')) {
-      setState(() {
```

Comment:
How are you updating the state here by removing ```setState(()){}```

Reply:
Please read this blog.
https://levelup.gitconnected.com/6-easy-tips-when-working-with-setstate-in-a-flutter-application-3d629bc77a16

Tip 3: Don’t call setState in initState methods

`initState` will trigger a rebuild after completion so it’s not necessary to call `setState` inside this method. This method is intended to initialize state-relevant properties like setting default values or subscribing to streams. Don’t do anything else here!